src/ontology/cl-edit.owl

```diff
@@ -27731,7 +27734,7 @@ EquivalentClasses(obo:CL_1001516 ObjectIntersectionOf(obo:CL_0000164 ObjectSomeV
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0010803") obo:IAO_0000115 obo:CL_1001517 "The various hormone- or neurotransmitter-secreting cells present throughout the mucosa of the stomach.")
 AnnotationAssertion(rdfs:label obo:CL_1001517 "stomach enteroendocrine cell")
```

Comment:
Please add more information for this term `stomach enteroendocrine cell` - (ie. references).
Also, the `stomach neuroendocrine cell `lacks a textual defs, references, synonyms etc.

This could be done in a separate ticket.

Reply:
Agree separate ticket